bi-connectors/PowerBIConnector/OpenSearchProject-ODBC.pq

```diff
@@ -1,18 +1,18 @@
 // This file contains Data Connector logic
 [Version = "1.0.0"]
-section OpenSearch;
+section #"OpenSearchProject-ODBC";
 
 // When set to true, additional trace information will be written out to the User log. 
 // This should be set to false before release. Tracing is done through a call to 
 // Diagnostics.LogValue(). When EnableTraceOutput is set to false, the call becomes a 
 // no-op and simply returns the original value.
 EnableTraceOutput = false;
 
-[DataSource.Kind="OpenSearch", Publish="OpenSearch.Publish"]
-shared OpenSearch.Contents = Value.ReplaceType(OpenSearchImpl, OpenSearchType);
+[DataSource.Kind="OpenSearchProject-ODBC", Publish="OpenSearchProject-ODBC.Publish"]
```

Comment:
Please repeat the removal of -ODBC except for the protocol.

Reply:
Addressed in https://github.com/Bit-Quill/opensearch-project-sql/pull/27/commits/f27402a988e667b46f62712520cd5337128a3008